content/en/tracing/llm_observability/quickstart.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Quickstart
+---
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+Our quickstart docs make use of our Python SDK. For detailed usage, see [the SDK documentation][1]. If your application is written in another language, you can create traces by calling the [API] instead.
+
+## Jupyter notebook quickstart
+
+To try out LLM Observability in a series of guided notebooks, follow the instructions in [the LLM Observability onboarding repo][2].
+
+## Command line quickstart
+
+Use the steps below to run a simple Python script that generates an LLM Observability trace.
+
+### Prerequisites
+
+- LLM Observability requires a Datadog API key (see [the instructions for creating an API key][7]).
+- The example script below uses OpenAI, but you can modify it to use a different provider. To run the script as written, you need:
+    - An OpenAI API key stored in your environment as `OPENAI_API_KEY`. To create one, see [Account Setup][4] and [Set up your API key][6] in the OpenAI documentation.
+    - The OpenAI Python library installed. See [Setting up Python][5] in the OpenAI documentation for instructions.
+
+### 1. Install the SDK
+
+Install the following `ddtrace` package hash:
+
+{{< code-block lang="shell" >}}
+pip install git+https://github.com/DataDog/dd-trace-py.git@c03cc8764c508cf53ea38b7b0f0eb6cb39627d91
```

Comment:
@lievan I meant to ask whether this install command should match the others, I figured this hash might be outdated by now but wasn't sure if the quickstart uses it for a specific reason.

Reply:
Ah thanks for catching this.

From now, can we change this (and all other hashes in the docs) to `<INSERT-DDTRACE-BETA-HASH>`? We'll probably either want a stable beta hash, or just tell the user to install @main, or a specific version of ddtrace -- will discuss this with the team and get back to you, but the placeholder will serve as a good reminder for now

Reply:
I saw that your PR is using @main, so I've set it to that just to avoid setting broken copy, since we're getting close to launch time. But let me know if any updates are needed!